src/Exceptions/HttpExceptionHtmlResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @var string $title
+ * @var string $phrase
+ * @var string $href
+ */
+?>
+<!DOCTYPE HTML>
+<html lang='en'>
```

Comment:
More single quotes in HTML

Single quotes for PHP
Double quotes for HTML

smh my head

Reply:
thats from the my IDE actually